distro/configs/openmrs/initializer_config/attributetypes/attribute_types.csv

```diff
@@ -1,3 +0,0 @@
-Uuid,Void/Retire,Entity name,Name,Description,Min occurs,Max occurs,Datatype classname,Datatype config,Preferred handler classname,Handler config,_order:1000
```

Comment:
Does this mean we'll be pulling the default attributes from openmrs?

Reply:
Yes.  These attributes were not specific to MSF and already defined [here](https://github.com/openmrs/openmrs-content-referenceapplication-demo/blob/main/configuration/backend_configuration/attributetypes/attribute_types-core_demo.csv).

The workflow is to pull from OpenMRS and add on content on top of it unless in special cases where we need to exclude some content for example the `locations`.  In such cases, we might need to redefine content that was already defined in OpenMRS